app/code/Magento/InventoryReservationCli/Model/SalableQuantityInconsistency/FilterExistingOrders.php

```diff
@@ -25,7 +25,7 @@ public function execute(array $inconsistencies): array
         return array_filter(
             $inconsistencies,
             function (SalableQuantityInconsistency $inconsistency) {
-                return (bool)$inconsistency->getOrder();
+                return $inconsistency->hasAssignedOrder() !== false;
```

Comment:
I believe we can get rid of `!== false` without any problem)

Reply:
done